test/ui-test/src/drivers/context.py

```diff
@@ -25,5 +25,3 @@ def detach(timeout_sec: int = configs.squish.PROCESS_TIMEOUT_SEC):
         ctx.detach()
         while ctx.isRunning and time.monotonic() - started_at < timeout_sec:
             time.sleep(1)
-                
-    utils.local_system.kill_process(configs.path.AUT.name)
```

Comment:
So this is no longer necessary?

Reply:
In the old framework, we start the application via squish.startApplication. It is a squish method. And we don't need to close it, because squish.detach(), which we are using at the end, should close the app.
Previously I added it just to be sure the app closed.
I will see on nightly regression result, and if necessary I will move it back.

Reply:
Very well.